string current_mission
string current_task
string engine_state
string description
mission_control/Function[] enabled_functions